src/traffic_logstats/logstats.cc

```diff
@@ -1275,6 +1277,19 @@ parse_log_buff(LogBufferHeader *buf_header, bool summary = false, bool aggregate
     bool agg = false;
     LogFormat::parse_symbol_string(buf_header->fmt_fieldlist(), fieldlist, &agg);
   }
+
+  // Validate the fieldlist
+  field                                                  = fieldlist->first();
+  constexpr std::array<std::string_view, 11> test_vector = {"cqtq", "ttms", "chi",  "crc", "pssc", "psql",
```

Comment:
Try putting more braces around the initializer. For some reason the CentOS and FreeBSD builds require this. See [here](https://github.com/SolidWallOfCode/trafficserver/blob/master/lib/ts/BufferWriterFormat.cc#L765).

Reply:
thx - trying it out.